### DISCUSSION TITLE
Cockroach DB migrations removes catalog tables

### DISCUSSION THREAD

**Discussion by jbmikk** [mikro-orm/mikro-orm#3102] [Q&A]
This is my configuration:
```
export default {
  type: 'postgresql' as const,
  driverOptions: {
    version: '13',
  },
  dbName: 'gdhistory',
  debug: true,
  user: 'root',
  host: 'cockroachdb',
  port: 26257,
  password: 'root',
  charset: 'utf8',
  entities: [ProjectEntity]
};
```

I created an empty database with:

    yarn mikro-orm database:create

Then I created the first migration file with:

    yarn mikro-orm migration:create

The migration file was created succesfully, but it contains `drop` commands for all the catalog tables that come with cockroach DB.
For instance:
```
    this.addSql('create table "project_entity" ("project_id" varchar(255) not null, "contents" jsonb not null);');
    this.addSql('alter table "project_entity" add constraint "project_entity_pkey" primary key ("project_id");');
    this.addSql('drop table if exists "crdb_internal"."active_range_feeds" cascade;');
    this.addSql('drop table if exists "crdb_internal"."backward_dependencies" cascade;');
    this.addSql('drop table if exists "crdb_internal"."builtin_functions" cascade;');
```

Is there a way of telling the migrations command to ignore everything in the `crdb_internal` schema?
Maybe I can force it to work on a specific schema or ignore tables with a specific pattern?

**Comment by B4nan** [comment]
Latest dev version of postgres driver ignores schemas prefixed with `crdb_` too and you can ignore additional schemas via schema generator options.
